app/javascript/src/PaymentGateways/braintree/BraintreeFormWrapper.jsx

```diff
@@ -0,0 +1,12 @@
+// @flow
```

Comment:
Not a big deal at all but I've been recently declaring the wrappers alongside their components. I may send a PR removing all our standalone "Wrappers".